website_sale_charge_payment_fee/controllers/main.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2018 Lorenzo Battistini - Agile Business Group
+# Copyright 2020 AITIC S.A.S
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import http
+from odoo.http import request
+
+from odoo.addons.website_sale.controllers.main import WebsiteSale
+
+
+class WebsiteSaleFee(WebsiteSale):
+    @http.route()
+    def payment(self, **post):
```

Comment:
This method doesn't seem to be called when I try selecting an acquirer in my local environment.